### PR TITLE
@mzikherman => Use a different on-the-fly Cloudfront url for Display Ads

### PR DESCRIPTION
**Pull request by kanaabe**
Thanks for helping + setting up the Cloudfront side! This PR makes use of the new url by adding an `isDisplay` option to the resizer methods.

### CODE REVIEW

**Comment by mzikherman** [comment]
`isDisplay` feels sort of generic/abstract for this (for example it doesn't refer to the underlying meaning- the separate handling of 'ads'). Maybe `isDisplayAd` or something is better?

**Comment by kanaabe** [comment]
Good idea! It makes sense in the Publishing realm since "display" is what we call "ads" but I think for general purposes `isDisplayAd` is much clearer. Updating...

**Comment by mzikherman** [comment]
Solid.

File: src/Utils/resizer.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
 import qs from "qs"
 const GEMINI_CLOUDFRONT_URL = "https://d7hftxdivxxvm.cloudfront.net"
+const GEMINI_DISPLAY_CLOUDFRONT_URL = "https://de5y2r7wr8mpb.cloudfront.net"
```
Comment (mzikherman) [inline src/Utils/resizer.ts]: I don't think it matters match currently, but these _might_ be better as ENV/config values, which clients should supply.

But, this is definitely easier for now, although these can definitely change/break in the future so it'd be nice be able to update apps easily in the future.

**Pull request merged**